drivers/sensor/ina23x/Kconfig

```diff
@@ -26,6 +26,15 @@ config INA237
 	help
 	  Enable driver for INA237.
 
+config INA237_VSHUNT
+	bool "INA237"
```

Comment:
It seems that it is copy&paste from the above. Please use a more suitable string.